libraries/AP_HAL_ChibiOS/hwdef/PixC4-Jetson/defaults.parm

```diff
@@ -0,0 +1,11 @@
+#Default Parameters for the Horizon31 PixC4-Jetson
+
+#Default setup for comms with companion computer
+SERIAL2_PROTOCOL 2
+SERIAL2_BAUD 500
+BRD_SAFETYENABLE 0
+FS_GCS_ENABLE 1
+#Generally, this system is not used with a traditional radio
+FS_THR_ENABLE 0
```

Comment:
this could be very dangerous. How does this system get pilot inputs?

Reply:
after review, agree.  removing default FS params and BRD_SAFETYENABLE